openquake/sep/liquefaction/vertical_settlement.py

```diff
@@ -25,44 +28,54 @@
 }
 
 
-
-def hazus_vertical_settlement(liq_susc_cat: Union[str, List[str]],
+def hazus_vertical_settlement(
+    liq_susc_cat: Union[str, List[str]],
+    pga: Union[float, np.ndarray],
+    mag: Union[float, np.ndarray],
     settlement_table: dict = HAZUS_VERT_SETTLEMENT_TABLE,
-    return_unit: str = 'm')-> Union[float, np.ndarray]:
+    return_unit: str = 'm',
+    groundwater_depth: float = 1.524,
+    do_map_proportion_correction: bool = True
+) -> Union[float, np.ndarray]:
     """
-    Distance of vertical settlement from Hazus 
-    (https://www.hsdl.org/?view&did=12760)
+    Distance of vertical settlement from Hazus.
 
     :param liq_susc_cat:
-        Liquefaction susceptibility category (LSC). This is a category denoting
-        the susceptibility of a site to liquefaction, independent of the
-        ground motions or earthquake magnitude. Acceptable values are:
-            `vh`: Very high
```

Comment:
We need the acceptable values for the LSC documented somewhere if we remove it from here.

Reply:
they are in sep_models.rst

Reply:
they are in sep_models.rst